tests/unit/test_server_cmds.py

```diff
@@ -258,7 +258,7 @@
                  'SNIA            Server                1.1.0',
                  'SNIA            Software              1.4.0'],
       'rc': 0,
-      'test': 'lines'},
+      'test': 'innows'},
```

Comment:
How is this change related to the purpose of this PR?

This comment also applies to all the other occurrences of this change in this file.

Reply:
We did this because the lines test is too detailed and we get caught in minor formatting issues that are not really part of the test.  I want to eliminate the lines test for all tests but those that absolutely require it.